src/pages/DashboardPage/Navbar/DropDown/DropDown.styled.ts

```diff
@@ -0,0 +1,74 @@
+import styled from "styled-components";
+
+export const Dropdown = styled.div`
+  padding-right: 10px;
```

Comment:
You are using on many places hard coded values, can you use var? [link](https://sass-lang.com/documentation/variables)